app/src/main/java/nu/yona/app/api/manager/dao/ActivityCategoriesDAO.java

```diff
@@ -75,7 +75,7 @@ public ActivityCategories getActivityCategories() {
                 }
             }
         } catch (Exception e) {
-            AppUtils.throwException(ActivityCategories.class.getSimpleName(), e, Thread.currentThread(), null);
+            AppUtils.reportException(ActivityCategories.class.getSimpleName(), e, Thread.currentThread(), null);
```

Comment:
Add an overload on ``reportException`` that doesn't need a listener, so you don't need to pass null all the time.

Reply:
Updated.